crates/samples/components/json_validator/src/lib.rs

```diff
@@ -316,8 +316,8 @@ fn sanitized_value() {
         );
         let sanitized = std::slice::from_raw_parts(sanitized_alloc, sanitized_len);
         let sanitized = String::from_utf8_lossy(sanitized);
```

Comment:
Maybe a comment like "This does not in fact return a `String`" or just something specific like `.into_owned()` to force the `String` to be allocated. Just want to leave a breadcrumb but I guess the PR does that so its probably fine as is.

Reply:
Happy to do that, esp. since it's a sample.